cm/proxy.c

```diff
@@ -575,11 +613,19 @@ M0_INTERNAL int m0_cm_proxy_remote_update(struct m0_cm_proxy *proxy,
 
 	M0_LOG(M0_DEBUG, "Sending to %s hi: ["M0_AG_F"]",
 	       proxy->px_endpoint, M0_AG_P(&in_interval->sw_hi));
+	if (proxy->px_send_final_update)
+		proxy->px_send_final_update = false;
```

Comment:
Can this be moved to the previous 'if' condition?

Reply:
Actually, I don't understand why the original code needs this.
I will try to remove this.

Reply:
Let's do as you suggested.